src/BowlingApp.js

```diff
@@ -0,0 +1,110 @@
+$(document).ready(function() {
+  var scorecard = new ScoreCard();
+
+  $("#play_again").click(function() {
+    $("#game_over").hide();
+    location.reload();
+  });
+
+  $("button").click(function(event) {
```

Comment:
As discussed, it would be great to break some of the stuff in here out to separate functions.